packages/components/src/circular-option-picker/types.ts

```diff
@@ -40,7 +40,16 @@ type CommonCircularOptionPickerProps = {
 	 * The child elements.
 	 */
 	children?: ReactNode;
-};
+} & (
+	| {
+			'aria-label': string;
+			'aria-labelledby'?: never;
+	  }
+	| {
+			'aria-label'?: never;
+			'aria-labelledby': string;
+	  }
+);
```

Comment:
This seems technically correct, but let's watch out for TS computed types since combining type "unions" can quickly result in a high number or resulting types

Reply:
First time for me familiarizing with 'unions' so any advice is more than welcome.
Also, there's a little bit or duplication in this PR, I'd appreciate any feedback about how to prevent it and abstract a bit.